spec/builders/online_application_builder_spec.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require 'rails_helper'
+
+describe OnlineApplicationBuilder do
+
+  let(:current_time) { Time.zone.now }
+  let(:submission) { attributes_for :public_app_submission }
+
+  describe '#build' do
+    subject(:build_submission) { described_class.new(submission).build }
+
+    it 'returns non persisted OnlineApplication' do
+      is_expected.to be_a(OnlineApplication)
+      is_expected.not_to be_persisted
+    end
+
+    describe 'email contact' do
```

Comment:
I suppose the real sanitisation and conversion will happen when we move the data from `OnlineApplication` to `Application` and then we need to do these various conversions.